scripts/ci-e2e.sh

```diff
@@ -48,27 +48,35 @@ export IMAGE_OS=${IMAGE_OS}
 export FORCE_REPO_UPDATE="false"
 export USE_IRSO="${USE_IRSO:-false}"
 EOF
-# if running basic integration or clusterctl-upgrade test skip apply bmhs in dev-env
-if [[ "${GINKGO_FOCUS:-}" == "clusterctl-upgrade" ]] || [[ "${GINKGO_FOCUS:-}" == "basic" ]] || [[ "${GINKGO_FOCUS:-}" == "integration" ]]; then
+
+case "${GINKGO_FOCUS:-}" in
+  'clusterctl-upgrade'|'k8s-upgrade'|'basic'|'integration')
```

Comment:
Quotes are not needed

Reply:
I was checking examples and they were using quotes. I can remove it not necessary